internal/nginx/templates/nginx-plus.tmpl

```diff
@@ -96,15 +94,15 @@ http {
 
         location  = /dashboard.html {
         }
-        {{ range $value := .NginxStatusAllowCIDRs }}{{ if ne $value "" }}
-        allow {{$value}};{{ end }}
-        {{end}}
+        {{range $value := .NginxStatusAllowCIDRs}}
```

Comment:
not possible for $value to be empty string right?

Reply:
Yeah, if `""` is provided as a CIDR, [`validateCIDRorIP`](https://github.com/nginxinc/kubernetes-ingress/blob/master/cmd/nginx-ingress/main.go#L370) returns an error and the controller [exits fatally](https://github.com/nginxinc/kubernetes-ingress/blob/master/cmd/nginx-ingress/main.go#L115) without executing the template.